lib/analytics/course_statistics.rb

```diff
@@ -35,7 +35,7 @@ def report_statistics
   end
 
   def articles_edited
-    Article.where(namespace: 0, id: @page_ids)
+    Article.where(namespace: 0, id: @article_ids)
```

Comment:
page_ids is correct here. In this file, `page_ids` is mw_page_ids, and `article_ids` is the ones that are namespace 0 (ie, not just pages but mainspace articles).